docs/backend-api/resources/tracking/tracker/index.md

```diff
@@ -1390,6 +1390,253 @@ For `tracker` object structure, see [tracker/](#tracker-object-structure).
 
 <hr>
 
+### replace
+
+Lets to replace the device without losing its history and some of its settings.
```

Comment:
I suggest adding a clarification on how this action is related to the registration process. It is not accidental that the words "register" and "registration" are constantly encountered further in the text. 
We know that "replace" action could be described as a registration of a new tracker object with attaching settings and data from existing tracker. Maybe it's right to  write directly about this similarity.

Reply:
done